crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -271,9 +273,18 @@ pub fn calculate_bounds(
 }
 
 pub fn update_frusta<T: Component + CameraProjection + Send + Sync + 'static>(
-    mut views: Query<(&GlobalTransform, &T, &mut Frustum)>,
+    mut views: Query<(
+        AnyOf<(&GlobalTransform, &GlobalTransform2d)>,
+        &T,
+        &mut Frustum,
+    )>,
```

Comment:
The system itself becomes less verbose with generics, but the scheduling gets really bad because of all the required `ambiguous_with` configuration.

Leaving this to a follow-up PR as these systems are not very performance sensitive.